docs/recipes/docker-server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14-buster-slim
+FROM node:16-buster-slim
```

Comment:
With node 14 in maintenance I feel moving to 16 is idea as it's in Active development still.